README.md

```diff
@@ -38,6 +38,7 @@ We run our DuckPAN tests against Perl 5.16 and 5.18 using Travis (https://travis
     + [Installing the Virtual Machine](#installing-the-virtual-machine)
     + [Using the Virtual Machine](#using-the-virtual-machine)
 - [Vagrant Virtual Environment](#vagrant-virtual-environment)
+    + [Setup Instructions](#setup-instructions-2)
```

Comment:
I was wondering about that myself. The old version had it. 3e09d6966208b73cb89cdbc77f93ad8c0f08057f
I will fix it though.